crates/compiler/test_gen/src/helpers/llvm.rs

```diff
@@ -572,47 +572,59 @@ pub fn try_run_lib_function<T>(
     }
 }
 
+// only used in tests
+#[allow(unused)]
+pub(crate) fn llvm_evals_to<T, U, F>(src: &str, expected: U, transform: F, ignore_problems: bool)
```

Comment:
now in a function instead of a macro. That gives more debug info, and also hopefully compiles faster than expanding a big macro 1000+ times